libs/blocks/aside/aside.css

```diff
@@ -7,9 +8,14 @@
   overflow: hidden;
 }
 
+.dark .aside {
+  color: white;
```

Comment:
This probs isn't necessary. Our global styles should catch any .dark > elem w/ this already.

Reply:
it looks like we are doing this in other blocks so leaving this until we create the global styles